dds/idl/dds_generator.cpp

```diff
@@ -332,11 +332,13 @@ string type_to_default(const std::string& indent, AST_Type* type, const string&
       if (use_cxx11) {
         def_val = "0.0L";
       } else {
+		    string temp_val = "ACE_CDR::LongDouble val = ACE_CDR_LONG_DOUBLE_INITIALIZER";
```

Comment:
Indent looks wrong... tabs?

Reply:
It doesn't look like this for me. I don't know how this is happening on github as it isn't what is on my branch.

Reply:
Nevermind, you are right there were tabs which were interpreted differently locally than on github.  Weird.

Reply:
Which I think is the main reason for not using tabs. Tabs are traditionally 8 spaces which Github is doing, but editors can be configured to display them differently.

Reply:
Okay, thank you.